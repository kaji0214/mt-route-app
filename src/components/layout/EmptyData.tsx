import { Box, Typography } from '@mui/material';
import React, { memo } from 'react';

const EmptyData = () => {
  return (
    <Box sx={{ width: '100%', maxWidth: 500 }}>
      <Typography variant="overline" display="block" gutterBottom>
        No Data
      </Typography>
    </Box>
  );
};

export default memo(EmptyData);
