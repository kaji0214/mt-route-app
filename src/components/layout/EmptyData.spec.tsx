import React from 'react';
import renderer from 'react-test-renderer';
import EmptyData from "./EmptyData";

it('renders correctly', () => {
  const tree = renderer
    .create(<EmptyData />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
