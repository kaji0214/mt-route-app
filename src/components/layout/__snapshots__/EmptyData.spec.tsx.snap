// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root css-gmbfhr"
>
  <span
    className="MuiTypography-root MuiTypography-overline MuiTypography-gutterBottom css-wxvrsw-MuiTypography-root"
  >
    No Data
  </span>
</div>
`;
